topcoffea/modules/corrections.py

```diff
@@ -21,13 +21,26 @@ def get_method1a_wgt_singlewp(eff,sf,passes_tag):
     wgt = p_data/p_mc
     return wgt
 
+def get_method1a_wgt_doublewp(effA, effB, sfA, sfB, cutA, cutB, cutC):
+    effA_data = effA * sfA
+    effB_data = effB * sfB
+
+    pMC = ak.prod(effA[cutA], axis=-1) * ak.prod(effB[cutB] - effA[cutB], axis=-1) * ak.prod(1 - effB[cutC], axis=-1)
+    pMC = ak.where(pMC==0,1,pMC) # removeing zeroes from denominator...
+    pData = ak.prod(effA_data[cutA], axis=-1) * ak.prod(effB_data[cutB] - effA_data[cutB], axis=-1) * ak.prod(1 - effB_data[cutC], axis=-1)
+
+    return pData, pMC
+
 # Evaluate btag sf from central correctionlib json
 def btag_sf_eval(jet_collection,wp,year,method,syst):
```

Comment:
I think when we added this function here it was already with correctionlib, right? I am wondering if you could say a bit about why these changes to `btag_sf_eval()` are needed?

Reply:
This looks like the method for multiple b-tagging WPs. Can you confirm @anpicci?